config/clock.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'clockwork'
+require 'active_support/time'
+
+def execute_rake_task(task)
+  `bundle exec rake #{task}`
+end
+
+module Clockwork
+  every(1.hour, 'withdrawals:run') { execute_rake_task('smartvpn:withdrawals:run') }
+
+  every(1.hour, 'courses:update') { execute_rake_task('smartvpn:courses:update') }
+
+  every(1.hour, 'proxy:update') { execute_rake_task('smartvpn:proxy:update') }
+
+  if `hostname`.strip == 'billing.smartvpn.biz'
```

Comment:
We can remove everything what is related to backup from billing.

Reply:
ok, I'll remove it.